config/management/operational/src/test_helper.rs

```diff
@@ -278,10 +286,12 @@ impl OperationalTool {
         &self,
         backend: &config::SecureBackend,
         disable_validate: bool,
+        disable_address_validation: bool,
     ) -> Result<(TransactionContext, Ed25519PublicKey), Error> {
         self.rotate_key(
             backend,
             disable_validate,
+            disable_address_validation,
```

Comment:
I think this is another instance where, basically we should only validate the addresses when someone specifically changes them?

Reply:
See my comment below.